### ISSUE TITLE
CSR enhancement

### ISSUE THREAD

**Issue by IlseFerris** [MN-Script-Team/DHS-MAXIS-Scripts#1041]
A member of my team today brought this up: Per CM, anyone who is not homeless, or a migrant or elderly/disabled needs a CSR. Example: GA clients need CSR for SNAP, but are IR exempt for HC.
*
SNAP units with earned, unearned, or no income who do NOT meet any of the criteria below are Six-Month Reporters:
●	Units in which all members are homeless. See HOMELESS in 0002.29 (Glossary: Gross RSDI...).
●	Units in which all members are in the migrant work stream. Not all members must be in agricultural work, but all members must be traveling together for this purpose.
●	Units in which any member is a seasonal farm worker.
●	Units in which all adult members are elderly or disabled if the unit has no earned income.
●	Units living on Indian reservations. Unit members do not have to be members of a tribe to be eligible for this exemption.
●	Those units receiving MFIP, including the Uncle Harry Food Support cases
*
Long story short, can we give the option to select both an exempt IR for HC AND SNAP (possibly in a drop down menu)?

**Comment by CDPotter** [comment]
Would we really need to be case noting this? There isn't any work to process for an exempt SNAP CSR while for the HC one there is an approval to be done/noted about.

**Comment by IlseFerris** [comment]
If someone is exempt from SNAP and HC, then yes, we should just used the 'cleared it' case note. But if someone is exempt from HC IR and is not exempt from SNAP, then the question is should they have the option to select that in the script.  Otherwise the other option is that we could put in a group box instructing/warning the worker that if the exempt IR checkbox is checked, that the 'cleared it' case note will be generated.  The instructions on SIR don't necessarily give instruction save for if it's exempt check the box, and we are seeing the 'cleared it' case note being used more and more as an agency instead of the detailed CSR case note.

**Comment by CDPotter** [comment]
Ah ok I totally misunderstood! (I forget that other counties don't have the paperless script) So in this situation they get the CSR and process both the SNAP CSR and the paperless IR and they shouldn't need to run the script twice to get the paperless IR clearing note. 

I think the CSR is currently glutted with objects otherwise I'd suggest just adding a checkbox for it. 

Maybe we could rework it so:
HC and IR checked = shortened note
SNAP only checked= full CSR note
SNAP and HC checked= full CSR note
SNAP and HC and IR checked = full CSR note but there's a line about paperless HC IR processed. 

Thoughts?

**Comment by IlseFerris** [comment]
That is exactly what I'm thinking ;D

Also we need to get the paperless IR out to the whole group soon.  Stupid red tape.

**Comment by theVKC** [comment]
I don't know that this policy as described above is correct. @ruthmaloney, care to weigh in?

**Comment by ruthmaloney** [comment]
It looks correct to me.
